docs/Questions.md

```diff
@@ -0,0 +1,98 @@
+<H1> Folder structure(Pages, Components, Templates, Organisms, Molecules, Atoms) </H1>
+We should stick to one way of organizing our folder and component's structure.
+Utils have files with .tsx file extension, which I think is inappropriate.
+We have validation.js and wdyr.js files at the top level of src
+
+<H1> Styles </H1>
+Currently we both use Styled-Components and scss across our codebase. I think we should stick to one and remove the other
+
+---
+
+Mike's comment: _Personally, I prefer styled-components because they allow you to pass variables to your styles from the component_
+
+<H1> Data fetching & selectors in components </H1>
+We have a lot of ambiguity about how we interact with data in our app.
+For example:
+
+- `useConnects` in components with different ways of solving the issue, when there is no needed data(like a venue id)
+- firestore direct pushes in `actions` and components
+
+<H1> Merging flow </H1>
+We should have a well-defined merging flow to avoid misunderstandings.
```

Comment:
Depending on the CI\CD strategy we choose, we could end up with 2 flows(normal | hotfix).

- Normal: feat|impr|fix branch => Staging => Production
- Hotfix: hotfix branch => Prod && hotfix branch => Staging

Reply:
I agree that it should be well defined.. i feel like I will probably have opinions on how this looks, but I don't have anything currently in mind.

My general preference with things like this is to look to existing standards/patterns, and evaluate what makes the most sense for us.

Do you have any examples of existing standards/patterns that you like, and why you like them (particularly, why you like them over other options)?

Reply:
See also: https://github.com/sparkletown/internal-sparkle-docs/pull/1#discussion_r579889314